data-serving/scripts/export-data/functions/05-aggregate/app.py

```diff
@@ -0,0 +1,278 @@
+import datetime
+import io
+import json
+import re
+import time
+
+import boto3
+import pandas as pd
+import pymongo
+import requests
+
+_JHU_COUNTRY_MAP = {
+    "Czechia": "Czech Republic",
+    "Korea South": "South Korea",
+    # there are cases in G.h under both Algiers and Algeria
+    "Algeria": "Algiers",
+    "US": "United States",
+    #     '': 'Gibraltar',
+    "Congo Kinshasa": "Democratic Republic of the Congo",
+    "Cote dIvoire": "Cote d'Ivoire",
+    "GuineaBissau": "Guinea-Bissau",
+    "Congo Brazzaville": "Republic of Congo",
+    #     '': 'Virgin Islands, U.S.',
+    #     '': 'Reunion',
+}
+
+_CODES_COUNTRY_MAP = {
+    "United States Virgin Islands": "Virgin Islands, U.S.",
+    "Congo": "Republic of Congo",
+    "Russian Federation": "Russia",
+    "United Republic of Tanzania": "Tanzania",
+    "Iran (Islamic Republic of)": "Iran",
+    "Algeria": "Algiers",
+    "Republic of Moldova": "Moldova",
+    "Viet Nam": "Vietnam",
+    "Bolivia (Plurinational State of)": "Bolivia",
+    "Republic of Korea": "South Korea",
+    "United Kingdom of Great Britain and Northern Ireland": "United Kingdom",
+    "Côte d'Ivoire": "Cote d'Ivoire",
+    "United States of America": "United States",
+    "Czechia": "Czech Republic",
+    "The former Yugoslav Republic of Macedonia": "North Macedonia",
+}
+
+_CODES_COUNTRY_ADD = {
+    "Kosovo": "XK",
+    "Reunion": "RE",
+    "Namibia": "NA",
+    "Taiwan": "TW",
+    "Faroe Islands": "FO",
+}
+
+_EXCLUDE = ["Puerto Rico"]
+
+username = "calremmel"
+password = "vVWD3bTFAXzlkpJ0"
```

Comment:
Should be in environment variables :)

Reply:
ohhh nooooooo. time to rotate these, classic 😭 😭